### PR TITLE
docs(firestore-bigquery-export): delete query

### LINKED ISSUE #444: firestore-bigquery-export: Add scheduled delete query for stale entries
### [READ] Step 1: Are you in the right place?

Issues filed here should be about a feature request for a **specific extension in this repository**. To file a feature request that affects multiple extensions or the Firebase Extensions platform, please reach out to
[Firebase support](https://firebase.google.com/support/troubleshooter/report/features/) directly.

### [REQUIRED] Step 2: Extension name

This feature request is for extension: **firestore-bigquery-export**

### What feature would you like to see?

It would be nice if when installing the extension, there is an option to create a scheduled query that deletes stale changelog entries. When querying the `latest` view, it queries the entire `changelog` table and this would help maintain query costs low. 
Plus, it's considered a "good idea" by [this blog post](https://cloud.google.com/blog/products/gcp/performing-large-scale-mutations-in-bigquery) from GCP.
### How would you use it?

The scheduled delete query would be optionally selected when installing the extension.

### PR DESCRIPTION
**Pull request by russellwheatley**
fixes #444 

Provide example of how to delete rows in the changelog table generated by the `firestore-bigquery-export` table

### CODE REVIEW

**Comment by i14h** [comment]
@kevinthecheung can you take a look at this?

**Review by dackers86 — APPROVED** [review]

**Comment by mvergarair** [comment]
I think the example given in the docs cloud delete entries in the changelog that are part of the current latest view. The recommended query cloud be something like delete all event ids that are not in the current latest view, and that are older than a day.

File: firestore-bigquery-export/guides/EXAMPLE_QUERIES.md [hunk 1]
```diff
@@ -103,8 +103,18 @@ If you had multiple arrays in the schema configuration, you might have to select
...
+```sql
 SELECT DISTINCT document_name, favorite_numbers_member
 FROM ${param:PROJECT_ID}.users.users_schema_user_full_schema_latest
 WHERE favorite_numbers_index = 0
 ```
+
+### Remove stale data from your changelog table
+
+If you wish to clean up the data from your `changelog` table, use the following `DELETE` query to provide
+a range of time against the `timestamp` column, and delete the requisite rows.
```
Comment (kevinthecheung) [inline firestore-bigquery-export/guides/EXAMPLE_QUERIES.md:115]:
```suggestion
If you want to clean up data from your `changelog` table, use the following
`DELETE` query to delete the rows that fall within a range of timestamps.
```

File: firestore-bigquery-export/guides/EXAMPLE_QUERIES.md [hunk 2]
```diff
@@ -103,8 +103,18 @@ If you had multiple arrays in the schema configuration, you might have to select
...
+
+```sql
+/* the first WHERE query is the start of the time range, the second WHERE query is the end of the time range. The below deletes every row in a 24 hour time frame on the 4th September, 2020*/
```
Comment (kevinthecheung) [inline firestore-bigquery-export/guides/EXAMPLE_QUERIES.md:118]:
```suggestion
/* The first WHERE clause is the start of the time range, and the second WHERE
   clause is the end of the time range. The query below deletes every row in a
   24-hour time frame on the 4th September, 2020. */
```

**Review by kevinthecheung — COMMENTED** [review]
Sorry I missed this!

**Comment by russellwheatley** [comment]
@mvergarair sounds interesting. Do you have a link for such a query? I'd be happy to update retrospectively.

**Review by kevinthecheung — APPROVED** [review]

**Review by dackers86 — APPROVED** [review]

**Comment by jhuleatt** [comment]
@mvergarair we're going to merge this as-is, but feel free to open a new issue to discuss an alternative cleanup query.

**Review by jhuleatt — APPROVED** [review]

**Pull request merged**